tests/test_harvest/test_mit_harvester.py

```diff
@@ -289,7 +290,7 @@ def test_mit_harvester_send_eventbridge_duplicate_record_sends_one_last_event(
         _output_records = list(harvester.send_eventbridge_event(iter(records)))
 
     mock_method.assert_called_once()
-    assert mock_method.mock_calls[0].args[2].source_record.event == "created"
+    assert not mock_method.mock_calls[0].args[2]["source_record_is_deleted"]
```

Comment:
Nice adjustment here!  I missed the `not` part at first, and wondered why it was working, but I understand now it's correctly the last event with `created` (and there _not_ `source_record_is_deleted`).

Reply:
This is one thing I really like about minitest in ruby that I haven't figured out how to do in pytest and was annoyed about recently when writing tests.

In ruby, `refute` is used instead of `assert not` which I think is much clearer when you are testing for a negative. The syntactic sure in ruby has meaningful differences in how we read code sometimes :)

Reply:
Totally.  Wish python did have a built-in `refute`.  

Did a bit of googling -- untested -- and it looks like maybe the closest would be something like this:

```python
import unittest

testcase = unittest.TestCase()
testcase.assertFalse(False)  # This will pass
testcase.assertFalse(True)   # This will fail
```

But... that's a lot of syntax.